twhite96/tiffany.blog
# Fixes anchor link hashes

This fixes the anchor link hashes issue originally reported in https://github.com/bryanbraun/anchorjs/issues/167.

There's also a commit which fixes the error in the console by removing the unnecessary request to a local (and nonfunctional) jQuery file.

**To test:**

1. Pull down the branch and run it locally: `jekyll serve`
2. Browse to the home page (`http://127.0.0.1:4000/`) and click on an AnchorJS link (next to the heading title).
3. Confirm that 1) the page smoothly scrolls the heading to the top of the page, and 2) the hash ID is added to the URL bar.
4. Copy the link in the URL bar, paste it into a new tab and load the page.
5. Verify that the page loads, and jumps to the ID shown in the URL.
6. Open the developer tools and verify that there are no console errors.
